packages/launch/src/hooks/use-domain-selection.ts

```diff
@@ -3,14 +3,15 @@
  */
 import * as React from 'react';
 import { useDispatch, useSelect } from '@wordpress/data';
-import type { DomainSuggestions } from '@automattic/data-stores';
+import { DomainSuggestions } from '@automattic/data-stores';
 import { mockDomainSuggestion } from '@automattic/domain-picker';
 import type { ResponseCartProduct } from '@automattic/shopping-cart';
+import { DOMAIN_SUGGESTIONS_STORE } from '../stores';
```

Comment:
We should join this import with the rest of the imports from `../stores` a few lines below